docs/datatype/image/image2image.md

```diff
@@ -85,19 +78,19 @@ class FlashImageEncoder(Executor):
             doc.embedding = embed.numpy()
 ```
 
-As one can see, how simple it is to build an `Encoder` executor. We simply inherit the base `Executor` and use decorator
-to define endpoints. As this `request` decorator is empty, it means that this function will be called regardless of the
-endpoints invoked, i.e., on both `/index` and `/search` endpoint. We
-leverage [lightning-flash](https://github.com/PyTorchLightning/lightning-flash) to use pre-trained `ResNet101` model for
-getting the embeddings. Reader can replace this model with any other pre-trained models of their choice. When this
-executor is instantiated, the pre-trained weights are downloaded automatically. The `predict` function takes in
-the `DocumentArray` and extracts embeddings which is then stored in the `embedding` attribute of the
+To build an Encoder Executor we inherit the base `Executor` and use a decorator
```

Comment:
```suggestion
To build an `Encoder` Executor we inherit the base `Executor` and use a decorator
```

Reply:
I don't think we need to backtick Encoder or Executor here. We're talking about them as concepts, not code